Makefile

```diff
@@ -138,25 +138,25 @@ cfssljson:
 certs: export PATH := $(shell pwd):$(PATH)
 certs: easy-rsa-master cfssl cfssljson
```

Comment:
We should be able to rename `easy-rsa-master` as well right? It seems like the package is just called `easy-rsa` (line 121)

Reply:
I can give it a go.